usagov/usagov-2021
# USAGOV-1369-A11Y-Banner

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1369

## Description

Add svg arrow to the banner. Remove old arrow that was not a11y. https://www.cms.gov/ and https://www.va.gov/ were given as examples on how new banner should behave. 

## Type of Changes

  - Add screenshot showing what it should look like {should look the same - no screenshot}

## Testing Instructions



### Change Requirements



### Validation Steps
On English and Spanish pages: 
Desktop: 

- Make sure banner functions the same (open/closes correctly should work with tab, spacebar or mouse)
- Inspect the banner and ensure the arrow is now an svg item in the DOM 
- Turn off images (on chrome go to [settings](chrome://settings), search images, open site settings , scroll to images, select "Don't allow sites to show images") 
- Ensure arrow is still there and functions correctly

Mobile: 
- Make sure banner functions the same (open/closes correctly should work with tab, spacebar or mouse)
- Change in mobile: previously the Here's how you know disappeared on accordion expansion. We don't want that anymore. Make sure it's still there when open. 
- Ensure close icon is there when accordion is open 
- Inspect the banner and ensure the arrow is now an svg item in the DOM 
- Turn off images (on chrome go to [settings](chrome://settings), search images, open site settings , scroll to images, select "Don't allow sites to show images") 
- Ensure arrow is still there and functions correctly


## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
